contributed_definitions/nyaml/NXcoordinate_system_set.yaml

```diff
@@ -88,138 +88,174 @@ doc: |
+        that is located behind that detector.
+      enumeration: [undefined, north, east, south, west, in, out]
+    y_direction(NX_CHAR):
+      doc: |
+        Direction of the positively pointing y-axis base vector of the processing_reference_frame.
+        We assume the configuration is inspected by looking towards the sample surface.
+        If a detector is involved, we assume the configuration is inspected from a position
+        that is located behind that detector.
+      enumeration: [undefined, north, east, south, west, in, out]
+    z_direction(NX_CHAR):
+      doc: |
+        Direction of the positively pointing z-axis base vector of the processing_reference_frame.
+        We assume the configuration is inspected by looking towards the sample surface.
+        If a detector is involved, we assume the configuration is inspected from a position
+        that is located behind that detector.
+      enumeration: [undefined, north, east, south, west, in, out]
+  sample_reference_frame(NXcoordinate_system):
+    doc: |
+      Details about the sample_reference_frame that is typically overlaid onto the surface of the sample.
+      We assume the configuration is inspected by looking towards the sample surface. If a detector
+      is involved, we assume the configuration is inspected from a position
+      that is located behind that detector.
+      
+      Reference DOI: 10.1016/j.matchar.2016.04.008 suggest to label the
+      base vectors of this coordinate system as Xs, Ys, Zs.
+    depends_on:
+      doc: |
+        Reference to the specifically named :ref:`NXsample` instance for
+        which these conventions apply (e.g. /entry1/sample1).
+    origin(NX_CHAR):
+      doc: |
+        Location of the origin of the sample_reference_frame.
+        
+        We assume regions-of-interest in this reference frame form a
+        rectangle or cuboid. Edges are interpreted by inspecting the
+        direction of their outer unit normals (which point either parallel
+        or antiparallel) along respective base vector direction of the
+        reference frame.
+      enumeration: [undefined, front_top_left, front_top_right, front_bottom_right, front_bottom_left, back_top_left, back_top_right, back_bottom_right, back_bottom_left]
+    x_direction(NX_CHAR):
+      doc: |
+        Direction of the positively pointing x-axis base vector of the sample_reference_frame.
+        We assume the configuration is inspected by looking towards the sample surface.
+        If a detector is involved, we assume the configuration is inspected from a position
+        that is located behind that detector.
+      enumeration: [undefined, north, east, south, west, in, out]
+    y_direction(NX_CHAR):
+      doc: |
+        Direction of the positively pointing y-axis base vector of the sample_reference_frame.
+        We assume the configuration is inspected by looking towards the sample surface.
```

Comment:
These repetitive statements could also just go to the docstring of sample_reference_frame, with a note that other assumptions are allowed (see above).

Reply:
Implemented following the suggestion